src/Api/Endpoints/Candidate/Candidate.php

```diff
@@ -0,0 +1,152 @@
+<?php declare(strict_types=1);
+/**
+ * This implements the Candidate endpoint class under Candidates.
+ *
+ * PHP Version 7
+ *
+ * @category API
+ * @package  Loris
+ * @author   Xavier Lecours Boucher <xavier.lecours@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris
+ */
+namespace LORIS\Api\Endpoints\Candidate;
+
+use \Psr\Http\Message\ServerRequestInterface;
+use \Psr\Http\Message\ResponseInterface;
+use \LORIS\Api\Endpoint;
+
+/**
+ * A class for handling the /candidates/$candidate endpoint.
+ *
+ * @category API
+ * @package  Loris
+ * @author   Xavier Lecours Boucher <xavier.lecours@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris
+ */
+class Candidate extends Endpoint implements \LORIS\Middleware\ETagCalculator
+{
+    /**
+     * The requested Candidate.
+     *
+     * @var \Candidate
+     */
+    private $_candidate;
+
+    /**
+     * Contructor
+     *
+     * @param \Candidate $candidate The requested candidate
+     */
+    public function __construct(\Candidate $candidate)
+    {
+        $this->_candidate = $candidate;
+    }
+
+    /**
+     * Return which methods are supported by this endpoint.
+     *
+     * @return array supported HTTP methods
+     */
+    protected function allowedMethods() : array
+    {
+        return array('GET');
+    }
+
+    /**
+     * Versions of the LORIS API which are supported by this
+     * endpoint.
+     *
+     * @return array a list of supported API versions.
+     */
+    protected function supportedVersions() : array
+    {
+        return array(
+                'v0.0.1',
```

Comment:
Is this still supported? I thought it got removed. I could very well be wrong though

Reply:
You are right, I will remove it.